lib/friendly_shipping/services/usps/parse_package_rate.rb

```diff
@@ -94,7 +94,8 @@ def call(rate_node, package, package_options)
 
             rate_value =
               if commercial_rate_requested_or_rate_is_zero && commercial_rate_available
-                rate_node.at(COMMERCIAL_RATE_TAG)&.text&.to_d || rate_node.at(COMMERCIAL_PLUS_RATE_TAG).text.to_d
+                commercial_rate = rate_node.at(COMMERCIAL_RATE_TAG)&.text.to_d
+                commercial_rate.zero? ? rate_node.at(COMMERCIAL_PLUS_RATE_TAG).text.to_d : commercial_rate
```

Comment:
are there already specs?

Reply:
👏

Reply:
It was originally a RuboCop failure which, when fixed, introduced a new bug which this change fixes. 😌